deepchecks/tabular/dataset.py

```diff
@@ -126,6 +127,23 @@ def __init__(
             raise DeepchecksValueError('Can\'t create a Dataset object with an empty dataframe')
         self._data = pd.DataFrame(df).copy()
 
+        # Validation - Check for duplicate columns
+        dataset_columns = self._data.columns
+        columns_frequency_counter = Counter(dataset_columns)
+
+        # Checking for duplicate columns
+        duplicated_columns = []
+        for column, frequency in columns_frequency_counter.items():
+            if frequency > 1:
+                duplicated_columns.append(column)
+                
+        if len(duplicated_columns) > 1:
```

Comment:
Otherwise it'll fail when there is only one duplicate (:
```suggestion
        if len(duplicated_columns) >= 1:
```

Reply:
Ok, I'll make the change.